crates/core/src/transport/connection_handler.rs

```diff
@@ -416,6 +438,9 @@ impl UdpPacketsListener {
                                 });
                             }
                             outbound_sym_key = Some(key);
+                            // now we need to send back a packet with our asymetric pub key for the remote to have
```

Comment:
Per other comments, the public key should be sent as part of the connect protocol - not in the transport layer.

Reply:
I completely disagree with this approach. It will just make everything more complicated.